### ISSUE TITLE
Documentation for V2?

### ISSUE THREAD

**Issue by WhatDoor** [Xeo786/Rufaydium-Webdriver#4]
Hi, 

from memory there used to be documentation for V2 and also a branch for it? But it seems to have been removed. Is there any plans for AHK v2 documentation? 

Also, I have been working with this library in V2 and I think there are some limitations with using CDP Fetch domain. I feel I could contribute towards the library if that is possible?

**Comment by Xeo786** [comment]
yes there was v2 branch until my account got hacked and I restore, Rufaydium was replaced with some other code, and when I restore Rufaydium I lost that v2 branch + stars, which was alpha version but I was working on v2 separately, 

you download the Latest V2 from the-Automator, it is free
[Rufaydium v2.0.3](https://www.the-automator.com/downloads/rufaydium-in-v2/)

it supports websocket and event handling, and removed the CDP class with Session.CDPCall() method

you can share your contribution on [AHK forum Post](https://www.autohotkey.com/boards/viewtopic.php?t=126189)

**Comment by WhatDoor** [comment]
Sorry to hear that you got hacked. Mainly I wanted to check about documentation because v1 documentation isnt always accurate to the new v2 code. I have downloaded it from the link you specified previously. 

Do you have plans to create a new repo for v2?

**Comment by dariovr1** [comment]
Hi @Xeo786 

Are any updates regarding the evolution of the library? A new releases are planned? Once the library is purchased on the-automator.com, the new update needs to be purchased again or a new free update will be released inside the purchased?

thank you.
